### ISSUE TITLE
Questions about ZymoResearch Quick DNA/RNA Microprep Plus Kit

### ISSUE THREAD

**Issue by yaaminiv** [RobertsLab/resources#1012]
I want to try the [ZymoResearch Quick DNA/RNA Microprep Plus Kit](https://github.com/RobertsLab/resources/blob/master/protocols/Commercial_Protocols/ZymoResearch_quick-dna-rna_microprep_plus_kit_20190411.pdf) with the Cv female/male gonad samples. I've [tried this kit on Cg adductor samples but got low DNA/RNA yield](https://yaaminiv.github.io/Gigas-Broodstock-RNA-Extraction/). I had some mess ups but even if I got double yields, it wouldn't have been enough for any WGBS/RNA-Seq applications.

Last time I tried the kit, I incubated non-homogenized tissue for 2-3 hours (....didn't actually put a specific time in my lab notebook but this is my guess based on the incubation times). Would I get better yields if I incubated non-homogenized tissue for 5 hours (suggested by kit), or if I homogenized tissue in liquid nitrogen then incubated? Is that something I just need to try myself? Any ideas @kubu4 ?

**Comment by kubu4** [comment]
Tissue should be homogenized in some fashion prior to lysing. At a minimum, tissue should be diced/minced prior to the lysing step.

You will get better yields with better homogenization, as well as with longer incubation times.

**Comment by yaaminiv** [comment]
Is there a such thing as "over incubation" for samples? I know the lysis step is just breaking open cells...does it stop when al the cells have been lysed? If I were to incubate samples overnight (even though the kit recommends no more than 30 minutes), what are the cons to that?

**Comment by kubu4** [comment]
The cons would be potential DNA degradation. Overnight is probably fine, though.

I don't think the kit indicates "no more than 30mins". The incubation timings chart is the recommended incubation times, based on their experience with each of those types of sample inputs, to achieve complete sample lysis.

**Comment by yaaminiv** [comment]
Got it! I'll try homogenizing with liquid nitrogen and an overnight incubation on a few samples to see what that does to yield.

**Comment by yaaminiv** [comment]
I used the kit today and had a decent DNA/RNA yield ([lab notebook](https://yaaminiv.github.io/Virginica-Gonad-DNA-Extractions-Part2/))! I have more than enough RNA but will need additional DNA for the 500 ng WGBS minimum. 

I had a question about extractions moving forward. I could continue using 5 mg of tissue each time, but I noticed that folks in Hollie's lab were homogenizing samples in the DNA/RNA shield. @hputnam Did you find that it improved your yield when you were using tissues? @kubu4 Could taking 5-10 mg of tissue, adding the DNA/RNA shield, and using the TissueTearor to break some of it down before an overnight incubation increase DNA yield?
